meeting-notes/2020-08-12-to-2020-12-16.md

```diff
@@ -0,0 +1,945 @@
+    -[Lore/Nova] Starting to add videos to falco.org 
+
+
+### Agenda
+
+- [Lore] What can we do to improve the community calls process
+    - We need an MC for next week: Spencer
+- [Lore] Release team for 0.26.0
+    - Leonardo Di Donato volunteered
+    - Leonardo Grasso voluntereed
+    - Kris Nova voluntereed
+    - Nova set up invite and sent to the mailing list for tomorrow 10am to 12pm Pacific
+    - Spencer volunteers for 0.27.0 team
+- [Lore/Leo] Talk about the eBPF runtime issues
+    - Working on a fix, does not load in > 5.8.5 kernels
+- [Leo/Lore] falco-driver-loader-improvements
+    - https://github.com/falcosecurity/falco/pull/1408
+- [Leo/Lore] driverkit improvements about ubuntu[-aws]
+    - https://github.com/falcosecurity/driverkit/pull/69
+    - https://github.com/falcosecurity/test-infra/pull/172
+    - Release v0.1.0
+- [Chichi Weng] [Falco overhead](https://github.com/falcosecurity/falco/issues/1404)
+    - For the disk write slowed down: @loris was suggesting that we can try to profile what happens in this scenario. However this looks like more an issue in the engine since not a lot of rules are hit in a situation like that. The CPU usage is related to the amount of events, this is a known limitation. We can probably improve the situation by doing filtering in the driver. Tl;dr: 24% percent is probably something to take into account if you want to run falco in a machine with that usage pattern.
+    - First experiment we can do is to put an UF_ALWAYS_DROP in https://github.com/draios/sysdig/blob/dev/driver/syscall_table.c#L59 to see how it behaves.
+    - Lorenzo to copy this context to the issue
+- [Chichi Weng] [cn_proc+netlink](https://github.com/falcosecurity/falco/issues/1397)
+    - Chichi shared their use case, as described in the issue
+        - Instead of monitoring every single syscall, only process, fork, exec are dealt with. This is faster but less complete.
+    - Loris suggests that since Falco is very generic in the inputs this can be added as another use case
+    - https://github.com/falcosecurity/falco/pull/1398/files
+    - Can be written as an userspace input, example here: https://github.com/fntlnz/userspace-example
+- [mstemm] [Falco Exceptions Proposal](https://github.com/falcosecurity/falco/pull/1376)
+- [Loris] Update on rules engine performance
+- [Fahad/MattCaya] Operational stability
+    - Bintray storage limit for pre-built drivers   (https://github.com/falcosecurity/falco/issues/1399)
+    - Kris to implement the last part of cleanup artifacts proposal: archive "old" drivers cleanup artifacts proposal to S3 and make falco-driver-loader lookup for them there too
+
+### Closing
+
+- **MC Next Call**: Spencer Krum
+- **Next Release**: (Lead: Kris Nova), (Help: Leonardo Di Donato) on 17th Sept
+
+---
+
+# Falco Community Call
+
+[More Information](https://github.com/falcosecurity/community)
+[Zoom Link](https://zoom.us/my/cncffalcoproject)
+[Recording]()
+
```

Comment:
```suggestion
```

We already have this in a single file (https://github.com/falcosecurity/community/blob/master/meeting-notes/2020-09-16.md).